src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -213,17 +213,7 @@ private static double accumulate(Double a, Double b) {
     }
 
     List<CompletableFuture<Void>> futures = new ArrayList<>();
-    futures.add(operateOnSubObjIfPresent(operation, line, ADJUSTMENT, httpClient, ctx, okapiHeaders, logger));
-    futures.add(operateOnSubObjIfPresent(operation, line, COST, httpClient, ctx, okapiHeaders, logger));
-    futures.add(operateOnSubObjIfPresent(operation, line, DETAILS, httpClient, ctx, okapiHeaders, logger));
-    futures.add(operateOnSubObjIfPresent(operation, line, ERESOURCE, httpClient, ctx, okapiHeaders, logger));
-    futures.add(operateOnSubObjIfPresent(operation, line, PHYSICAL, httpClient, ctx, okapiHeaders, logger));
-    futures.add(operateOnSubObjIfPresent(operation, line, SOURCE, httpClient, ctx, okapiHeaders, logger));
-    futures.add(operateOnSubObjIfPresent(operation, line, VENDOR_DETAIL, httpClient, ctx, okapiHeaders, logger));
     futures.addAll(operateOnSubObjsIfPresent(operation, line, ALERTS, httpClient, ctx, okapiHeaders, logger));
-    futures.addAll(operateOnSubObjsIfPresent(operation, line, CLAIMS, httpClient, ctx, okapiHeaders, logger));
-    futures.addAll(operateOnSubObjsIfPresent(operation, line, FUND_DISTRIBUTION, httpClient, ctx, okapiHeaders, logger));
-    futures.addAll(operateOnSubObjsIfPresent(operation, line, LOCATIONS, httpClient, ctx, okapiHeaders, logger));
     futures.addAll(operateOnSubObjsIfPresent(operation, line, REPORTING_CODES, httpClient, ctx, okapiHeaders, logger));
```

Comment:
I just realized this, based on the comments in https://github.com/folio-org/acq-models/pull/89, Reporting codes are supposed to be po line independent, as this method is also called during po line deletion, we are potentially deleting reporting code too while deleting the po line. Reporting codes should be omitted while deleting

Reply:
Discussed in person yesterday with @javalkar-varun...  Good catch @KVupp.  The intended purpose of reporting codes is very similar to that of tags.  I suspect that we might opt to just support tags and remove reporting codes altogether at some point, but for now we need to keep them.